File: app/src/main/java/com/islam/basepropject/ui/Main2Activity.java
package com.islam.basepropject.ui;

import android.os.Bundle;

import com.islam.basepropject.R;
import com.islam.basepropject.project_base.base.POJO.NavigationType;
import com.islam.basepropject.project_base.base.activities.BaseNavigationActivity;
import com.islam.basepropject.ui.ExFetchData.Fragment1;
import com.islam.basepropject.ui.ExViewPager.Fragment4;

public class Main2Activity extends BaseNavigationActivity {

    @Override
    public void onLaunch() {
        initContentView(R.layout.activity_main2);
        initNavigation(new Class[]{Fragment1.class, Fragment2.class, Fragment4.class},
                new int[]{R.id.navigation_home, R.id.navigation_dashboard, R.id.navigation_notifications},
                NavigationType.BottomNavigation);    }


}

File: app/src/main/java/com/islam/basepropject/ui/Main2Activity.kt
package com.islam.basepropject.ui

import com.islam.basepropject.R
import com.islam.basepropject.project_base.POJO.NavigationType
import com.islam.basepropject.project_base.base.activities.BaseNavigationActivity
import com.islam.basepropject.ui.ExFetchData.Fragment1
import com.islam.basepropject.ui.ExViewPager.Fragment4

class Main2Activity : BaseNavigationActivity() {
    override val layoutId = R.layout.activity_main2

    override val navigationType = NavigationType.BottomNavigation

    override val menuIdsListWithFragment: MutableList<Pair<Int, Class<*>>>
        get() = mutableListOf<Pair<Int, Class<*>>>().run {
            add(Pair(R.id.navigation_home, Fragment1::class.java))
            add(Pair(R.id.navigation_dashboard, Fragment2::class.java))
            add(Pair(R.id.navigation_notifications, Fragment4::class.java))
            this
        }
}
